.github/workflows/update-readme.yml

```diff
@@ -17,13 +17,15 @@ jobs:
         run: |
           sudo apt-get update
           sudo add-apt-repository universe
-          wget https://github.com/neovim/neovim/releases/download/nightly/nvim.appimage
+          wget https://github.com/neovim/neovim/releases/download/v0.5.0/nvim.appimage
           chmod u+x nvim.appimage
           mkdir -p ~/.local/share/nvim/site/pack/nvim-treesitter-textobject/start
           ln -s $(pwd) ~/.local/share/nvim/site/pack/nvim-treesitter-textobject/start
           mkdir -p ~/.local/share/nvim/site/pack/nvim-treesitter/start
           cd ~/.local/share/nvim/site/pack/nvim-treesitter/start
           git clone https://github.com/nvim-treesitter/nvim-treesitter.git
+          cd nvim-treesitter
+          git checkout 0.5-compat
```

Comment:
Why checkout 0.5 compat? It will automatically checkout the current branch. PR should be tested and not current 0.5-compat

Reply:
I didn't changed this file, it's from an older commit. I think it's the only diff of 0.5-compat and master and since we merge it from 0.5-compat to master git appended this commit to this pr.

Reply:
I see. Yes this was my fault then. Can you rebase your changes on master then? `git rebase -i origin/master` where origin is our remote. You'll probably need to drop the commit that added this.

Do you want me to fix this?

Reply:
Yes i think it's better you fix this. Thanks for your help & patience.

Reply:
That's great! You then fixed three issues in this PR. Not bad. Thank you!